l1-contracts/contracts/bridgehub/IBridgehub.sol

```diff
@@ -60,6 +60,18 @@ interface IBridgehub is IL1AssetHandler {
 
     event SettlementLayerRegistered(uint256 indexed chainId, bool indexed isWhitelisted);
 
+    /// @notice Emitted when the bridging to the chain is started.
+    /// @param chainId Chain ID of the hyperchain
+    /// @param assetId Asset ID of the token for the hyperchain's STM
+    /// @param settlementLayerChainId The chain id of the settlement layer the chain migrates to.
+    event BridgeBurn(uint256 indexed chainId, bytes32 indexed assetId, uint256 indexed settlementLayerChainId);
```

Comment:
I think it would make sense to switch the order of chainId and settlementLayerChainId, as we have a similar named function in IL1AssetHandler. Maybe similarly for BridgeMint

Reply:
Discussed in private, the rename of the event is enough